metadrive/component/pgblock/create_pg_block_utils.py

```diff
@@ -72,6 +72,11 @@ def CreateRoadFrom(
     Usage: give the far left lane, then create lane_num lanes including itself
     :return if the lanes created cross other lanes
     """
+    center_line_type = center_line_type or PGLineType.CONTINUOUS
+    side_lane_line_type = side_lane_line_type or PGLineType.SIDE
```

Comment:
Repetitive `side_lane_line_type = side_lane_line_type or PGLineType.SIDE`. 

By the way, it seems that this implementation is similar but more robust to invalid values, compared to the previous implementation which defines the defualt value in the function declaration. Is this what you want? Or is there are other reasons for this?

Reply:
Yes of course. We should not define the defalut value in function declariation. This is because in many case the function's argument will be passed in as None (the most convinent way for the outer caller to specify "oh I want the default value")